JPS_Ontology/ontology/ontoreaction/OntoReaction.owl

```diff
@@ -931,6 +1052,6 @@
 
 
 
-<!-- Generated by the OWL API (version 5.1.0.2017-03-29T23:31:42Z) https://github.com/owlcs/owlapi/ -->
+<!-- Generated by the OWL API (version 5.5.0.2023-01-14T12:41:38Z) https://github.com/owlcs/owlapi/ -->
```

Comment:
Was this updated in the JPS_BASE_LIB pom file?

Reply:
I am not sure. Do I need to update the pom file?